src/porepy/models/momentum_balance.py

```diff
@@ -638,14 +639,14 @@ def interface_displacement(self, interfaces: list[pp.MortarGrid]) -> pp.ad.Varia
         )
 
     def contact_traction(self, subdomains: list[pp.Grid]) -> pp.ad.Variable:
-        """Fracture contact traction.
+        """Fracture contact traction [-].
 
         Parameters:
             subdomains: List of subdomains where the contact traction is defined. Should
                 be of co-dimension one, i.e. fractures.
 
         Returns:
-            Variable for fracture contact traction.
+            Variable for nondimensionalized fracture contact traction.
```

Comment:
For clarity: Is the thought that all quantities in PorePy will get nondimensionalized?

Reply:
I was hoping you would answer that question!